testenv/docker/krb5kdc-older/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN yum install -y \
 
 ADD krb5-1.6.3.tar.gz /tmp
 RUN cd /tmp/krb5-1.6.3/src && \
-  ./configure && make && make install
+  ./configure && make CFLAGS="-fgnu89-inline" && make install
```

Comment:
Fixes the inlining/duplication issue with recent versions of `gcc` compiler:
👉🏻  https://bugzilla.redhat.com/show_bug.cgi?id=547153